tools/lint/tests/test_lint.py

```diff
@@ -213,6 +213,16 @@ def test_ref_absolute_url(caplog):
     assert "ref/absolute.html" in caplog.text
 
 
+def test_about_blank_as_ref(caplog):
+    with _mock_lint("check_path") as mocked_check_path:
+        with _mock_lint("check_file_contents") as mocked_check_file_contents:
+            rv = lint(_dummy_repo, ["about:blank"], "normal")
```

Comment:
You don't want `about:blank` here (that'll just try and lint a file called `about:blank`); you want to add a file to `tools/lint/tests/dummy/ref` with a `<link rel=match href=about:blank>` in it (see the other files in that directory), and then you'll need to add a second file for the other test mentioned above.